docs/publisher/index.rst

```diff
@@ -84,11 +84,23 @@ If you need different batching settings, simply provide a
     from google.cloud.pubsub import types
 
     client = pubsub.PublisherClient(
-        batch_settings=types.BatchSettings(max_messages=500),
+        batch_settings=types.BatchSettings(
+                                            max_messages=500, # default 100
+                                            max_bytes=1024, # default 1 MB
```

Comment:
Just making sure this should be 1 MiB.

Reply:
No, that was an error, it should be 1MB, not MiB